net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
To be more consistent with everything else in the packages feed I added an option enabled instead, as "git grep config_get_bool disabled" returned nothing. I set the default in the init script to 1, and disable it in the default config file. In this case, if the user removes the line it will work. Ok for you?

Maybe we should think of standardizing this in /etc/rc.common, so that all packages can use this without having to check it in their init script.